README.markdown

```diff
@@ -40,11 +39,13 @@ ini_setting { "sample setting":
 ## Usage
 
 
-The inifile module tries hard not to manipulate your file any more than it needs to. In most cases, it doesn't affect the original whitespace, comments, ordering, etc.
+The inifile module is used to: 
 
- * Supports comments starting with either '#' or ';'.
- * Supports either whitespace or no whitespace around '='.
- * Adds any missing sections to the INI file.
+ * Support comments starting with either '#' or ';'.
+ * Support either whitespace or no whitespace around '='.
+ * Add any missing sections to the INI file.
+ 
+It does not manipulate your file any more than it needs to. In most cases, it doesn't affect the original whitespace, comments, ordering, etc. See the common usages below for examples.
```

Comment:
Good change, much better. I would also suggest getting rid of 'etc." We shouldn't use Latin abbreviations like that, generally.